OpenTreeMap/src/OTM/OTMAppDelegate.m

```diff
@@ -118,4 +121,18 @@ - (BOOL) configureRollbarWithEnvironment:(OTMEnvironment*)env
     }
 }
 
+- (BOOL) configureGoogleAnalytics:(OTMEnvironment*)env
+{
+    NSString *trackingId = [env appGoogleAnalyticsId];
+    if (trackingId) {
+        [[GAI sharedInstance] setDryRun:YES];
```

Comment:
No, good catch. But since it's there perhaps I'll leave it for @jwalgran's benefit and remove it before merging.